Wireframe/css/style.css

```diff
@@ -0,0 +1,64 @@
+#git{
+    
+    text-align: center;
+    display: block;
+    width: auto;
+    height: auto;
+    margin-top: 30px;
+    margin-bottom: 30px;    
+}
+
+section:first-child{
+    border:2px solid blue;
+    width:fit-content;
+    display: flex;
+    margin-left: auto;
+    margin-right: auto;
+    justify-content: center;
+    
+    }
+
+ #hero{
```

Comment:
Try and format your code nicely before submitting.  Get into the habit of having everything nicely lined up.  VS Code will do that for you automatically every time you save your file (configure Prettier to do that).